api/src/certification/session-management/domain/usecases/supervise-session.js

```diff
@@ -2,26 +2,41 @@ import {
   CertificationCenterIsArchivedError,
   InvalidSessionSupervisingLoginError,
   SessionNotAccessible,
-} from '../../domain/errors.js';
+} from '../errors.js';
 
 const superviseSession = async function ({
   sessionId,
   invigilatorPassword,
   userId,
+  i18n,
   sessionRepository,
   supervisorAccessRepository,
   certificationCenterRepository,
 }) {
   // should use a specific get from sessionRepository instead
   const session = await sessionRepository.get({ id: sessionId });
 
+  if (session.isAccessBlocked()) {
+    const formattedDate = new Intl.DateTimeFormat('fr-FR').format(new Date(session.pixCertifBlockedAccessDate));
+    const translate = i18n.__;
+    throw new SessionNotAccessible(
+      translate('certification.error-messages.pix-certif-access', {
+        date: formattedDate,
+      }).replaceAll('&#x2F;', '/'),
+      'PIX_CERTIF_ACCESS_BLOCKED',
+    );
+  }
```

Comment:
Comme présenté lors de la MeJ sur la _gestion des locales_, l’équipe Accès recommande désormais de ne plus passer `i18n` aux usecases, mais de passer la locale aux usecases et de faire l’appel `const i18n = getI18n(locale);` dans les usecases. Ce qui donnerait : 

```suggestion
const superviseSession = async function ({
  sessionId,
  invigilatorPassword,
  userId,
  locale,
  sessionRepository,
  supervisorAccessRepository,
  certificationCenterRepository,
}) {
  // should use a specific get from sessionRepository instead
  const session = await sessionRepository.get({ id: sessionId });

  if (session.isAccessBlocked()) {
    const formattedDate = new Intl.DateTimeFormat(locale).format(new Date(session.pixCertifBlockedAccessDate));
    const translate = i18n.__;
    throw new SessionNotAccessible(
      translate('certification.error-messages.pix-certif-access', {
        date: formattedDate,
      }).replaceAll('&#x2F;', '/'),
      'PIX_CERTIF_ACCESS_BLOCKED',
    );
  }
```

Nous n’avons pas encore appliqué cette recommandation dans le code mais ce sera très bientôt, alors autant que ça soit déjà le cas ici 😃